cmd/minikube/cmd/env.go

```diff
@@ -55,6 +64,19 @@ func buildDockerEnvShellOutput(envMap map[string]string) string {
 	return output
 }
 
+func buildUnsetShellOutput() string {
+	envArr := []string{"DOCKER_TLS_VERIFY", "DOCKER_HOST", "DOCKER_CERT_PATH"}
+	output := ""
+	for _, envName := range envArr {
+		output += fmt.Sprintf("unset %s\n", envName)
```

Comment:
That definitely makes sense.  I will store the initial values on the primary docker-env call to the .minikube directory and then I will use the store value to populate the unset command.  Does this sound like a better/more-useful idea?  Let me know, thanks!